File: rest_api/src/main/java/com/longdrink/rest_api/model/payload/LoginWeb.java
package com.longdrink.rest_api.model.payload;

public class LoginWeb {
    private String nombreUsuario;
    private String contrasena;
    private String email;
    private String nombreCompleto;

    public LoginWeb(){}

    public LoginWeb(String nombreUsuario, String contrasena, String email, String nombreCompleto) {
        this.nombreUsuario = nombreUsuario;
        this.contrasena = contrasena;
        this.email = email;
        this.nombreCompleto = nombreCompleto;
    }

    public String getNombreUsuario() {
        return nombreUsuario;
    }

    public void setNombreUsuario(String nombreUsuario) {
        this.nombreUsuario = nombreUsuario;
    }

    public String getContrasena() {
        return contrasena;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getNombreCompleto() {
        return nombreCompleto;
    }

    public void setNombreCompleto(String nombreCompleto) {
        this.nombreCompleto = nombreCompleto;
    }
}

File: android_app/app/src/main/java/com/longdrink/androidapp/model/LoginWeb.kt
package com.longdrink.androidapp.model

class LoginWeb {

    private lateinit var nombreUsuario : String
    private lateinit var contrasena : String
    private lateinit var email : String
    private lateinit var nombreCompleto : String
    private lateinit var rol : String

    constructor(nombreUsuario: String, contrasena: String, email: String, nombreCompleto: String, rol: String) {
        this.nombreUsuario = nombreUsuario
        this.contrasena = contrasena
        this.email = email
        this.nombreCompleto = nombreCompleto
        this.rol = rol
    }
}